packages/appstore/src/modules/wallets/wallets.scss

```diff
@@ -20,6 +20,13 @@
 
         @include mobile {
             padding: 2.4rem 0;
+            gap: 1.6rem;
         }
     }
+
+    &__button-toggle {
```

Comment:
@aizad-deriv This style is not related to `wallets.tsx`, Please scope it to the component that uses it 🙏🏻